prow/config/config.yaml

```
@@ -238,7 +238,27 @@ branch-protection:
                   - "chunks (10)"
                   - "tso-function-test"
                   - "idc-jenkins-ci/build"
-                strict: true                           
+                strict: true
+            release-6.6:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "DCO"
+                  - "tide"
+                  - "statics"
+                  - "chunks (1)"
+                  - "chunks (2)"
+                  - "chunks (3)"
+                  - "chunks (4)"
+                  - "chunks (5)"
+                  - "chunks (6)"
+                  - "chunks (7)"
+                  - "chunks (8)"
+                  - "chunks (9)"
+                  - "chunks (10)"
+                  - "tso-function-test"
+                  - "idc-jenkins-ci/build"
+                strict: true                            
```

```suggestion
                strict: false                            
```

please set to `false` for batch merging